client/my-sites/checklist/wpcom-checklist/index.jsx

```diff
@@ -2,571 +2,57 @@
-				duration={ translate( '%d minute', '%d minutes', { count: 10, args: [ 10 ] } ) }
-				onClick={ this.handleTaskStart( {
-					task,
-					tourId: 'checklistPublishPost',
-					url: taskUrls.post_published,
-				} ) }
-				onDismiss={ this.handleTaskDismiss( task.id ) }
-				title={ translate( 'Publish your first blog post' ) }
-			/>
-		);
-	};
-
-	renderCustomDomainRegisteredTask = ( TaskComponent, baseProps, task ) => {
-		const { translate, siteSlug } = this.props;
-
-		return (
-			<TaskComponent
-				{ ...baseProps }
-				bannerImageSrc="/calypso/images/stats/tasks/custom-domain.svg"
-				completedButtonText={ translate( 'Change' ) }
-				completedTitle={ translate( 'You registered a custom domain' ) }
-				description={ translate(
-					'Memorable domain names make it easy for people to remember your address — and search engines love ’em.'
-				) }
-				duration={ translate( '%d minute', '%d minutes', { count: 2, args: [ 2 ] } ) }
-				onClick={ this.handleTaskStart( {
-					task,
-					tourId: 'checklistDomainRegister',
-					url: `/domains/add/${ siteSlug }`,
-				} ) }
-				onDismiss={ this.handleTaskDismiss( task.id ) }
-				title={ translate( 'Register a custom domain' ) }
-			/>
-		);
-	};
-
-	renderMobileAppInstalledTask = ( TaskComponent, baseProps, task ) => {
-		const { translate } = this.props;
-
-		return (
-			<TaskComponent
-				{ ...baseProps }
-				bannerImageSrc="/calypso/images/stats/tasks/mobile-app.svg"
-				completedButtonText={ translate( 'Download' ) }
-				completedTitle={ translate( 'You downloaded the WordPress app' ) }
-				description={ translate(
-					'Download the WordPress app to your mobile device to manage your site and follow your stats on the go.'
-				) }
-				duration={ translate( '%d minute', '%d minutes', { count: 3, args: [ 3 ] } ) }
-				onClick={ this.handleTaskStartThenDismiss( {
-					task,
-					url: '/me/get-apps',
-					dismiss: true,
-				} ) }
-				onDismiss={ this.handleTaskDismiss( task.id ) }
-				title={ translate( 'Get the WordPress app' ) }
-			/>
-		);
-	};
-
-	renderSiteLaunchedTask = ( TaskComponent, baseProps, task ) => {
-		const { translate } = this.props;
-
-		return (
-			<TaskComponent
-				{ ...baseProps }
-				bannerImageSrc="/calypso/images/stats/tasks/launch.svg"
-				buttonText={ translate( 'Launch site' ) }
-				completedTitle={ translate( 'You launched your site' ) }
-				description={ translate(
-					'Your site is private and only visible to you. Launch your site, when you are ready to make it public.'
-				) }
-				onClick={ this.handleLaunchSite( task ) }
-				onDismiss={ this.handleTaskDismiss( task.id ) }
-				title={ translate( 'Launch your site' ) }
-			/>
-		);
-	};
-}
-
-function getContactPage( posts ) {
-	return find(
-		posts,
-		post =>
-			post.type === 'page' &&
-			( some( post.metadata, { key: '_headstart_post', value: '_hs_contact_page' } ) ||
-				post.slug === 'contact' )
-	);
+		return <AsyncLoad require="./component.jsx" { ...ownProps } />;
```

Comment:
I recommend to use the "canonical" import name: `require="my-sites/checklist/wpcom-checklist"`. The import name is used to generate the webpack chunk name. Which is now `async-load-component-jsx.js`

Reply:
Oh, didn't know about that. Done in #28678.